examples/jsm/controls/CarControls.d.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  Object3D,
+} from '../../../src/Three';
+
+export class CarControls {
+  constructor( maxSpeed: number, acceleration: number, brakePower: number, turningRadius: number, keys: object );
```

Comment:
All ctor parameters are optional so you can write them like so `maxSpeed?: number`.

Reply:
OK, done.